src/main/java/ee/carlrobert/codegpt/toolwindow/chat/ui/textarea/UserPromptTextArea.java

```diff
@@ -194,11 +219,70 @@ public void actionPerformed(@NotNull AnActionEvent e) {
         }));
     if (isImageActionSupported()) {
       iconsPanel.add(new IconActionButton(new AttachImageAction()));
+      if (!ApplicationManager.getApplication().isUnitTestMode()) {
+        setDropTarget(new DropTarget() {
+          @Override
+          public synchronized void dragEnter(DropTargetDragEvent evt) {
+            isDragActive = true;
+            var t = evt.getTransferable();
+            var isSupportedFile = false;
+            try {
+              List<File> files = (List<File>) t.getTransferData(
```

Comment:
Hmm, apparently this results in `null` on macOS, or at least it doesn't work for me. 😞 

Here's bit of a context: https://stackoverflow.com/a/64926775

I tried using the latest JB JRE while running the plugin locally, as suggested. I also installed the plugin manually on my actual IDE, but both scenarios resulted in a NPE. I haven't tried the first option yet tho.

Reply:
Hm that is quite annoying. Unfortunately I do not have macOS so I can't try it out myself 🙃